test/test.py

```diff
@@ -562,6 +565,18 @@ def test_large_bundle(self):
                                                   version='2019-09-26T054644.254919Z')
         self.assertEqual(755, len(manifest))
 
+    def test_submitter_id(self):
```

Comment:
This was a bit of a comprehensive change as many tests were using `_test_bundle()` as well as the redundancy that was factored out of `_test_example_bundle()`. I've made this change as a separate commit in case we want to rethink this refactor.
Note: the refactor also removed the import of `humancellatlas.data.metadata.helpers.schema_examples.download_example_bundle` that appears to no longer exist in the repo.